### ISSUE TITLE
keyboard close after each chip is add

### ISSUE THREAD

**Issue by saadbzu** [tylersuehr7/chips-input-layout#50]
When i write and press enter on keyboard chip add and keyboard hide , please help how implement that the keyboard remains open?

**Comment by Ryanhustler420** [comment]
Yes that's the default behaviour! you can use `setDelimiter()` method but, unfortunately that's not present anywhere in the source code. :'(
